src/data/h1/tags/weapon.yml

```diff
@@ -389,36 +389,61 @@ comments:
       en: ...
       fields:
         - name: flags
-          en: ...
+          en: Various flags that change the behavior of the given trigger.
           fields:
             - name: tracks fired projectile
               en: ...
             - name: random firing effects
-              en: ...
+              en: >
+                If false the firing effects in the firing effects block will be
+                picked in order. If true they will be picked randomly.
             - name: can fire with partial ammo
-              en: ...
+              en: >
+                The weapon will be able to fire as long as the magazine content is
+                above `0`.
             - name: does not repeat automatically
-              en: ...
+              en: >
+                Disables repeat fires while holding the trigger. If true you need
+                to release the trigger before it can fire again.
+                This breaks if the weapon is a charge weapon like the plasma pistol/
+                when the player is a client in multiplayer it will start charging
+                again as soon as it is fired. (This bug is only noticeable if
+                `discharging spews` is set to true).
             - name: locks in on off state
-              en: ...
+              en: >
+                Makes the trigger behave like a switch. Tap it to turn it on. Tap
+                it again to turn it off.
             - name: projectiles use weapon origin
-              en: ...
+              en: >
+                If true the projectiles will spawn at the third person primary trigger
+                or secondary trigger marker. Instead of originating from the center
+                of the first person view.
```

Comment:
Sentences shouldn't start with "instead" unless you plan to introduce the alternative later in the sentence (e.g. "Instead of X, do Y"). You can just do "...secondary trigger marker, instead of..." instead.